core/lane.py

```diff
@@ -2240,6 +2246,50 @@ def works_from_database(self, _db, facets=None, pagination=None, **kwargs):
 
         return qu
 
+    def _restrict_query_for_no_hold_collections(
+        self, _db: Session, qu: query.Query
+    ) -> query.Query:
+        """Restrict the query to not select works that are part of a collection
+        that dissallow holds through DONT_DISPLAY_RESERVES if they have no available copies
+        This is only meant for Bibliotheca collections, but since they're the only ones with
+        the DONT_DISPLAY_RESERVES setting it will implicitly only be active for them
+        """
+
```

Comment:
@tdilauro is there any time when we don't want this functionality to be active? Should this be configurable via its __init__ args?

Reply:
We would always want it available in all library associations to Bibliotheca collections.

Reply:
Ok, this should take care of that then